tests/unit/rpc/handlers/SubscribeTests.cpp

```diff
@@ -604,16 +602,16 @@ TEST_F(RPCSubscribeHandlerTest, StreamsWithoutLedger)
         })"
     );
     runSpawn([&, this](auto yield) {
-        auto const handler = AnyHandler{SubscribeHandler{backend, subManager_}};
+        auto const handler = AnyHandler{SubscribeHandler{backend, mockSubscriptionManagerPtr}};
+        EXPECT_CALL(*mockSubscriptionManagerPtr, subTransactions).Times(1);
```

Comment:
Minor thing, feel free to ignore: `Times(1)` is by default when `EXPECT_CALL()` used, so you don't need to specify it. But it's ok to leave it as it is.